pkg/k8s/watchers/watcher.go

```diff
@@ -891,6 +894,12 @@ func (k *K8sWatcher) K8sEventReceived(apiResourceName, scope, action string, val
 	k.k8sResourceSynced.SetEventTimestamp(apiResourceName)
 }
 
+// K8sServiceEventProcessed is called to do metrics accounting the duration to program the service.
+func (k *K8sWatcher) K8sServiceEventProcessed(action, scope string, startTime time.Time) {
+	duration := time.Now().Sub(startTime)
```

Comment:
We should use the time.Since() api here.

Reply:
Replaced with `safetime.TimeSinceSafe`